src/react/databrowser/objects/ObjectDelete.jsx

```diff
@@ -83,8 +83,8 @@ const ObjectDelete = ({ bucketName, toggled, prefixWithSlash }: Props) => {
                 <Table>
                     <T.Body>
                         {
-                            toggled.map(s => (
-                                <T.Row key={s.name}>
+                            toggled.map((s, index) => (
+                                <T.Row key={`${s.name}-${index}`}>
```

Comment:
Why is this needed? `s.name` should be unique.

Reply:
I saw a warning about a non unique key from this line

Reply:
We should try to avoid `index` when we can. https://reactjs.org/docs/lists-and-keys.html#keys